glide.yaml

```diff
@@ -0,0 +1,9 @@
+package: kubevirt.io/kubevirt-ansible
+import:
+- package: github.com/onsi/ginkgo
+  version: 11459a886d9cd66b319dac7ef1e917ee221372c9
+- package: github.com/onsi/gomega
+  version: dcabb60a477c2b6f456df65037cb6708210fbb02
```

Comment:
stupid question, do we have to ginkgo & gomega here? cant we rely on the versions from kubevirt?

Reply:
KubeVirt does not specify it under `glide.yaml` file, so I prefer to explicitly set it here.